source/AI.cpp

```diff
@@ -1978,7 +1978,15 @@ void AI::PrepareForHyperspace(Ship &ship, Command &command)
 	bool isJump = (ship.GetCheapestJumpType(ship.GetTargetSystem()).first == Ship::JumpType::JumpDrive);
 
 	Point direction = ship.GetTargetSystem()->Position() - ship.GetSystem()->Position();
-	if(!isJump && scramThreshold)
+	const double squaredDeparture =  isJump ?
```

Comment:
Let's overshoot a little bit by adding some (named) constant; we don't want AI ships to "hang at the jump border" because they move to a point that just doesn't pass the check on line 1984 by some tiny value.

Reply:
done